### PR TITLE
Don't look at Components_to_build in deploy

### PR DESCRIPTION
**Pull request by krdln**

### CODE REVIEW

File: CIScripts/Deploy/Deployment.ps1 [hunk 1]
```diff
@@ -17,15 +17,18 @@ function Install-Artifacts {
     Param ([Parameter(Mandatory = $true)] [PSSessionT] $Session,
            [Parameter(Mandatory = $true)] [string] $ArtifactsDir)
 
+    function Test-NonemptyDir {
```
Comment (m-kostrzewa) [inline CIScripts/Deploy/Deployment.ps1:20]: IMHO this should be moved to Common/ dir

Comment (krdln) [reply CIScripts/Deploy/Deployment.ps1:20]: I don't feel like creating a separate file for a one-liner function. Is there any file I can add it to?

**Review by jablonskim — APPROVED** [review]
MJ & MC

File: CIScripts/Deploy/Deployment.ps1 (hunk 1 above)
Comment (m-kostrzewa) [reply CIScripts/Deploy/Deployment.ps1:20]: I guess we'll see if some other part of the codebase requires this function.

**Pull request merged**